.circleci/config.yml

```diff
@@ -0,0 +1,215 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/lockbox-addon
+    docker:
+      - image: circleci/node
+    steps:
+      - checkout
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@6.7'
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: install-npm
+          command: npm install
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+  test:
+    docker:
+      - image: circleci/node
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@6.7'
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+      - run:
+          name: install-npm
+          command: npm install
+      - save_cache:
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+      - run:
+          name: Install test dependencies
+          command: |
+            GECKODRIVER_URL=$(
+              curl -s 'https://api.github.com/repos/mozilla/geckodriver/releases/latest' |
+              python -c "import sys, json; r = json.load(sys.stdin); print([a for a in r['assets'] if 'linux64' in a['name']][0]['browser_download_url']);"
+            );
+            curl -L -o geckodriver.tar.gz $GECKODRIVER_URL
+            gunzip -c geckodriver.tar.gz | tar xopf -
+            chmod +x geckodriver
+            sudo mv geckodriver /bin
+            geckodriver --version
+      - run:
+          name: Install Firefox
+          command: |
+            sudo npm i -g get-firefox
+            get-firefox --branch nightly --platform linux --extract --target ~/
+            ~/project/firefox -v
+            firefox --version
```

Comment:
if I'm reading the above command correctly ...

1. `get-firefox` is extracts/instals to `~/`
2. attempts to run firefox to get its version from `~/project/firefox` (which doesn't exist)
3. attempts to run system-installed firefox to get its version